test/channel/base_test.rb

```diff
@@ -92,6 +92,11 @@ def rm_rf
     assert_nil @channel.last_action
   end
 
+  test "try calling a public method defined on Base" do
+    @channel.perform_action 'action' => :unsubscribe_from_channel
+    assert_nil @channel.last_action
```

Comment:
Now that you opened my eyes it's pretty hard to do it :) Will add a real test for methods defined on `Base`